src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -360,10 +362,11 @@ protected final void verify(Checker checker,
             message.append("differing violations: ").append(differingViolations);
         }
 
-        assertTrue(message.toString(),
-                missingViolations.isEmpty()
+        assertWithMessage(message.toString())
+                .that(missingViolations.isEmpty()
                         && unexpectedViolations.isEmpty()
-                        && differingViolations.isEmpty());
+                        && differingViolations.isEmpty())
+                .isTrue();
```

Comment:
I want a more fluent approach to this. Any suggestions?

Reply:
Maybe 
```java
assertWithMessage(message.toString())
    .that(missingViolations)
    .isEmpty();
assertWithMessage(message.toString())
    .that(unexpectedViolations)
    .isEmpty();
// so on
```
?

Reply:
Done this. However, I was looking for something more abstract.

Reply:
I think it makes it clear now.